### PR TITLE
Installing devise

### PR DESCRIPTION
**Pull request by paul-ot**
Trello Card: https://trello.com/c/KKf2y6HE/9-instalar-configurar-devise-y-crear-user

Details:
I have created this branch based on branch: 8-configurar-rubocop, so I'm creating this PR to such branch, please let me know if that is not correct and I'll change the base branch.

I have added Devise gem and I've created a new Devise model 'User', with two additional fields, first_name and last_name, both of them are not nullable and required.

Thanks!

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+  # ==> Configuration for any authentication mechanism
+  # Configure which keys are used when authenticating a user. The default is
+  # just :email. You can configure it to use [:username, :subdomain], so for
+  # authenticating a user, both parameters are required. Remember that those
+  # parameters are used only when authenticating and not when retrieving from
+  # session. If you need permissions, you should implement that in a before filter.
+  # You can also supply a hash where the value is a boolean determining whether
+  # or not authentication should be aborted when the value is not present.
+  # config.authentication_keys = [:email]
```
Comment (holywyvern) [inline config/initializers/devise.rb]: Remove commented code

File: db/migrate/20190204131055_devise_create_users.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.inet     :current_sign_in_ip
```
Comment (holywyvern) [inline db/migrate/20190204131055_devise_create_users.rb]: Same here, remove unused commented code

**Review by holywyvern — CHANGES_REQUESTED** [review]

File: config/initializers/devise.rb (hunk 1 above)
Comment (paul-ot) [reply config/initializers/devise.rb]: Done.

File: db/migrate/20190204131055_devise_create_users.rb (hunk 2 above)
Comment (paul-ot) [reply db/migrate/20190204131055_devise_create_users.rb]: Done.

**Comment by paul-ot** [comment]
@holywyvern, could you review my changes?
Thanks!

**Review by mtejedorwolox — APPROVED** [review]

**Review by holywyvern — APPROVED** [review]

**Pull request merged**